app/Architecture/ExceptionHandler.php

```diff
@@ -13,4 +16,19 @@ class ExceptionHandler extends Handler
         "password",
         "password_confirmation",
     ];
+
+    public function render($request, Throwable $e): Response
+    {
+        $response = parent::render($request, $e);
+
+        if (!app()->environment(["local", "testing"]) && in_array($response->status(), [500, 503, 429, 419, 404, 403, 401], true)) {
```

Comment:
Why only these statuses? Plus it would be nice to move this array to some const maybe?

Reply:
And why not testing? It would be nice to have a way to test failed operations.

Reply:
These statuses are rendered by default by Laravel. The solution itself is proposed by the creator of Inertia [link](https://inertiajs.com/error-handling), so I think that will be waste of time to make something more complicated. I've changed `environment` to production only. Maybe we can think about the application sidebar when the user is signed in, but it will be here some technical complications and it could be confusing to users.